spec/form_models/offender_sar_case_form_spec.rb

```diff
@@ -1,125 +0,0 @@
-require 'rails_helper'
-
-RSpec.describe OffenderSARCaseForm, type: :model do
-  let(:case_form) { OffenderSARCaseForm.new({}) }
-
-  it 'can be created' do
```

Comment:
I still think it would be good to keep this spec rather than delete it completely - even if it is just to keep `it can be created`.

There are still two public methods on the `OffenderSarCaseForm.rb` which I think could be tested.

A wider point is that it makes sense to keep this file even with minimal tests as it serves as a scaffold for future changes to `OffenderSarCaseForm.rb`. The fact that there is a `spec` file present makes people thinking that changes they make should have associated tests. If no test file exists there is an 'effort' barrier to creating one again.